### PR TITLE
Feat:"Archive Page"

### PR DESCRIPTION
**Pull request by sadathshariff**
Implemented Archive Page
Move to Archive
Restore from Archive
Delete From Archive

### CODE REVIEW

File: src/context/Note/utils.js [hunk 1]
```diff
@@ -51,3 +52,59 @@ export const updateNote = async (note, noteDispatch) => {
...
+    if (res.status === 200) {
+      noteDispatch({
+        type: "DELETE_FROM_ARCHIVE",
+        payload: res.data.archives,
+      });
+      Toast("Deleted from Archives", "success");
+    }
+  } catch (error) {
+    console.log("DElete", error);
+    Toast("Couldn't Delete from Archive ", "error");
```
Comment (raman1999) [inline src/context/Note/utils.js:105]: remove console

**Review by raman1999 — APPROVED** [review]
code looks clean  liked the way you are breaking down each component

**Review by aishwarya-bm — APPROVED** [review]
Everything looks good, i like separate utils file for your notes actions

**Review by RahulSingh9131 — APPROVED** [review]
Code contains perfect use of created context , easy to read and understandable. Go ahead and merge..👍👍

**Pull request merged**